### PR TITLE
set trust domain aliases to include both new Hub WIP and old Hub WIP to achieve no downtime upgrade for VM

### PR DESCRIPTION
**Pull request by jtrbs**

### CODE REVIEW

**Comment by williamaronli** [comment]
from cloud run https://pantheon.corp.google.com/cloud-build/builds;region=global/403ce2ee-6546-4317-b796-ae450e45eb56;step=13?project=asm-scriptaro-oss

there seems a test error 
``` 
Applying WorkloadGroup vm in namespace c65d258316a98fd2...
Error from server (MethodNotAllowed): error when creating "STDIN": create not allowed while custom resource definition is terminating
All Istio resources will be pruned from the cluster
```

**Comment by williamaronli** [comment]
@myidpt could you also take a look at this PR if that is ok to merge after tests have passed

File: scripts/asm-installer/install_asm [hunk 1]
```diff
@@ -2522,6 +2522,12 @@ configure_package() {
     kpt cfg set asm anthos.servicemesh.istiod-vs-name "istiod-vs-${REVISION_LABEL}"
   fi
 
+  if [[ "${CA}" == "mesh_ca" && "${USE_VM}" -eq 1 ]]; then
```
Comment (myidpt) [inline scripts/asm-installer/install_asm:2525]: Can we remove && "${USE_VM}"? The aliasing is needed when using mesh CA in general.

Comment (williamaronli) [reply scripts/asm-installer/install_asm:2525]: Ok I think so

Comment (jtrbs) [reply scripts/asm-installer/install_asm:2525]: Sure

**Comment by williamaronli** [comment]
close this one and create a new pr to resolve the issue https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/732/files

**Pull request closed without merging**